Product Management Service/Code/product_management_service.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE products (
+  id int PRIMARY KEY,
+  name varchar(255),
+  price decimal(10,2),
+  category varchar(255) FOREIGN KEY REFERENCES category(product_category)
+);
+
+CREATE TABLE product_details (
+  id int PRIMARY KEY,
+  image blob,
+  description text
+);
+
+CREATE TABLE product_category (
+  category varchar PRIMARY KEY,
+  specification JSON 
```

Comment:
why should **specification** be a part of product_category? I think this is because we want category-wise specifications? I think this coupling is not necessary. We can keep this unique to all products. So, this can be a part of product_details itself.